bidswitch-json/src/main/scala/com/powerspace/openrtb/bidswitch/BidSwitchSerdeModule.scala

```diff
@@ -18,11 +18,18 @@ object BidSwitchSerdeModule extends SerdeModule {
   implicit val customConfig: Configuration = Configuration.default.withSnakeCaseMemberNames
 
   /**
-    * BidSwitch bid response decoders
+    * BidSwitch bid request encoders
     */
-  override implicit val bidDecoder: Decoder[SeatBid.Bid] = BidSwitchBidSerde.bidDecoder
-  override implicit val seatBidDecoder: Decoder[SeatBid] = BidSwitchBidSerde.seatBidDecoder
-  override implicit val bidResponseDecoder: Decoder[BidResponse] = BidSwitchBidResponseSerde.decoder
+  override implicit val userEncoder: Encoder[BidRequest.User] = BidSwitchUserSerde.encoder
```

Comment:
Remove explicit types into the whole Pull Request plz `Encoder[BidRequest.User]` they convey no useful information

Reply:
I do not agree, I like having types. I am not a human compiler